### PR TITLE
Move logout link to the welcome banner

### PR DESCRIPTION
**Pull request by evsheffield**
Super small change to move the Logout link to the banner. I also ditched the little "down arrow" next to the user name because it felt misleading and with Logout in the banner there were too many arrows.

### CODE REVIEW

File: next/components/welcome-banner.js [hunk 1]
```diff
@@ -75,11 +71,16 @@ export default function WelcomeBanner() {
...
-            {user.username || 'Welcome'}
-          </a>
-        </Link>
+        <div>
+          <Link href="/account">
+            <a className="welcome-banner-link welcome-banner-link-username">
+              {user.username || 'Welcome'}
+            </a>
+          </Link>
+          <Link href="/auth/logout">
```
Comment (jtrussell) [inline next/components/welcome-banner.js]: This one should just be a regular ol' anchor tag since there's no next page for it.

Comment (jtrussell) [reply next/components/welcome-banner.js]: It'll still work with `Link`, we'll just try to load a JS bundle for a client side render before giving up and falling back on an SSR.

Comment (jtrussell) [reply next/components/welcome-banner.js]: (and I know I'm the guy who put it initially with a `Link` and no inner `a` at all 🤣)

Comment (evsheffield) [reply next/components/welcome-banner.js]: Hah I didn't even notice that, thanks! `Link` is just so pleasing to read...

**Pull request merged**